app/controllers/StoriesVisibleController.scala

```diff
@@ -11,57 +12,11 @@ case class StoriesVisibleRequest(
   stories: Seq[Story]
 )
 
-object StoriesVisibleResponse {
-  implicit val jsonFormat = Json.format[StoriesVisibleResponse]
-}
-
-case class StoriesVisibleResponse(
-  desktop: Option[Int],
-  mobile: Option[Int]
-)
-
-class StoriesVisibleController(val containers: Containers, val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) {
+class StoriesVisibleController(val containerService: ContainerService, val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) {
```

Comment:
The logic here now lives in a service, and the controller is nice and slim as a result.